middleware/src/main/java/br/ufrn/dimap/middleware/identification/lookup/Lookup.java

```diff
@@ -32,6 +37,17 @@
 	 * @param name	Name that'll be used to find the absolute object reference, 
 	 * 				previously binded with this same name.
 	 * @return		The absolute object reference binded with the given name.
+	 * @throws RemoteError
 	 */
-	public AbsoluteObjectReference find(String name) throws RemoteError;;
+	public AbsoluteObjectReference find(String name) throws RemoteError;
+	
+	/**
+	 * Allow the server requestor find the remote object reference given an 
+	 * object id.
+	 * 
+	 * @param objectId Unique identifier of remote object.
+	 * @return Correspondent of this object id.
+	 * @throws RemoteError
+	 */
+	public Object findById(ObjectId objectId) throws RemoteError;
```

Comment:
This method should really return an Object instance? What about returning an AOR (like the other find method)